mrtt-ui/src/components/CausesOfDeclineForm.js

```diff
@@ -288,21 +287,21 @@ function CausesOfDeclineForm() {
                             (secondaryChildOption, secondaryChildIndex) => {
                               return (
                                 <ListItem key={secondaryChildIndex}>
-                                  <NestedFormSectionDiv>
-                                    <Checkbox
-                                      value={secondaryChildOption}
-                                      onChange={(event) =>
-                                        handleCausesOfDeclineOnChange({
-                                          event,
-                                          mainCauseLabel: mainCause.label,
-                                          subCauseLabel: subCause.secondaryLabel,
-                                          secondaryChildOption
-                                        })
-                                      }></Checkbox>
-                                    <Typography variant='subtitle2'>
-                                      {secondaryChildOption}{' '}
-                                    </Typography>
-                                  </NestedFormSectionDiv>
+                                  <FormControlLabel
```

Comment:
I noticed this when I noticed that your reusable NestedFormSectionDiv is easily confused with my new NestedQuestion. Tried to figure out which one to use, and then realized yours could be solved with FormControlLabel. So took the liberty to change it.